skins/Seasons/hilo.inc

```diff
@@ -40,7 +40,10 @@
             $archive[1].outTemp.min.format(add_label=False)</span>
         </td>
         #end for
-        <td class="units">$unit.label.outTemp</td>
+        <td class="units">
```

Comment:
not sure what your intent is with these extra unit entries.  the units are already specified for each observation.

Reply:
I think it makes sense to have every value with his unit, it's easier to read. But it's my opinion